terraform/environments/electronic-monitoring-data/transfer_server_capita.tf

```diff
@@ -159,8 +159,8 @@ resource "aws_iam_role_policy" "capita_transfer_user_iam_policy" {
 # Set the public ssh key for the supplier user profile to access SFTP server.
 #------------------------------------------------------------------------------
 
-resource "aws_transfer_ssh_key" "capita_ssh_key" {
+resource "aws_transfer_ssh_key" "capita_ssh_key_ecdsa_sha2_nistp384" {
   server_id = aws_transfer_server.capita_transfer_server.id
   user_name = aws_transfer_user.capita_transfer_user.user_name
-  body      = "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQCqXTKqBVVBQX5lvCdCdws4t7lCVaniv3FGCaJQOKMYAzBzwcVD9MKz0RzH7FMMA/iBayw/+13Mb79paBkJdT8T/Wg9lER/YE/lPKZcyT2IJ6myW5kDShQAY9lQliRoJ4oVx9x95hGx48eE9jWsCtwEaQT7pH2aK5l2THqfFCDQEMmT84CaSmJzuxsaYxuohlVcMqnGdU/oq+E76gLm3Z0gvh3NwFHd0RTIzqlVgwEUbTcHqZBON522229VypuvqfIcD9WIPEMnza/rA/6FX5luniqh+h/PCF7HH3Qiveui3PZV64fQtqd2pVnK8llW7CLjXKC1/TkWx1QkWyGzGYBZUXEctNbOBMixFcVbj49CucWMztPC88gZl2bHlJPqdBLMt6sakigCLWJLIvB/oeXGhzCN7XkfKWXDTu4mHuQ+UHPbXzsPvPRxidfxxRVk758M+GB15nQq2Fm3lRtYgZ2mnjQT7dwhhCaiqJiy0qs5kQ4Hs9Jnex6afoPQlqrhamtu8= test.user@K9999"
+  body      = "ecdsa-sha2-nistp384 AAAAE2VjZHNhLXNoYTItbmlzdHAzODQAAAAIbmlzdHAzODQAAABhBIhggGYKbOk6BH7fpEs6JGRnMyLRK/9/tAMQOVYOZtehKTRcM5vGsJFRGjjm2wEan3/uYOuto0NoVkbRfIi0AIG6EWrp1gvHNQlUTtxQVp7rFeOnZAjVEE9xVUEgHhMNLw=="
```

Comment:
Is it worth maybe adding this via AWS secretmanager?

Reply:
See https://github.com/ministryofjustice/modernisation-platform-environments/pull/4503#discussion_r1449061694 for an example of how to do this.

Reply:
have reached out to security though.

Reply:
Response from cybersec on using secret manager for the public keys
> On the face of it, there is no reason to keep public keys private as public keys are intended to be public (that is how the PKI is designed and how https websites work).  Is there a reason for wanting to do this?  Perhaps all other secrets and parameters are kept in AWS SM so a desire to store in one place? 

With that in mind will stick with storing in terraform.